packages/ember-toucan-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@crowdstrike/ember-toucan-core",
-  "version": "0.1.0-beta.0",
+  "version": "0.1.0-beta.1",
```

Comment:
The version is actually a bit weird IMHO: I would say we do either `1.0.0-beta.x` releases, or just normal 0.x prereleases!

Reply:
Sure, yeah I'm good with either!  I think maybe a preference to `0.x` prereleases.  I think bots will also successfully pick up on version updates when they're pre-1.0.0, while they don't with the `beta` tag.  What do you think? I think we are quite a ways off from `1.0.0`, so if we are okay with a high numbered `1.0.0-beta.{big-number}` that's fine.

Reply:
After merging #130 this has been updated automatically to release a "normal" 0.1.0 version, so seems good to me! I'll leave to honour to merge this and push the release to either @nicolechung or @ynotdraw then! 😜